gameApi/index.ts

```diff
@@ -2,15 +2,15 @@ import { BigNumber, ethers } from 'ethers';
 import { createSessionWallet, getSessionWallet } from 'helpers/session_signatures';
 import { IGameMove, ISignedGameMove } from "../types/arbiter";
 import { TContractGameState } from 'components/Games/types';
-import { GameProposedEventObject, GameStartedEventObject } from "../.generated/contracts/esm/types/polygon/Arbiter";
+import { Arbiter, GameProposedEventObject, GameStartedEventObject } from "../.generated/contracts/esm/types/polygon/Arbiter";
 import { getPolygonSdk} from "../.generated/contracts";
 import { TGameType } from 'types/game';
 import { fetchSigner } from '@wagmi/core';
 
 const provider = new ethers.providers.AlchemyProvider('matic', process.env.NEXT_PUBLIC_ALCHEMY_API_KEY!);
 const getSdk = async () => getPolygonSdk(await getSigner())
 const getSdkRead = () => getPolygonSdk(provider);
-export const getArbiter = async () => (await getSdk()).arbiter;
+export const getArbiter: ()=>Promise<Arbiter> = async () => (await getSdk()).arbiter;
```

Comment:
it is simpler to set only the returning type, not to describe the full type of a function: 
getArbiter = async (): Promise<Arbiter> => {...}

Reply:
done